openam-server/pom.xml

```diff
@@ -89,6 +90,7 @@
                     </archive>
                     <container>
 	                    <containerId>tomcat9x</containerId>
+	                    <type>embedded</type>
```

Comment:
It seems there is an issue when using cargo's Embedded Container. 
It does not pass jvm args such as `--add-opens` and `--add-exports` to support JDK 8 compartibility. 
This happens, because the embedded container runs in the current JVM (https://codehaus-cargo.github.io/cargo/Embedded+Container.html). 
So OpenDJ can't generate self signed certificate and setup process fails. I suggest to revert to default Installed Container